build-config/scripts/onie-mk-iso.sh

```diff
@@ -100,6 +100,11 @@ RECOVERY_EFI_BOOTX86_IMG="$RECOVERY_EFI_BOOT_DIR/bootx64.efi"
 RECOVERY_ELTORITO_IMG="$RECOVERY_ISO_SYSROOT/boot/eltorito.img"
 RECOVERY_EMBEDDED_IMG="$RECOVERY_DIR/embedded.img"
 RECOVERY_UEFI_IMG="$RECOVERY_ISO_SYSROOT/boot/efi.img"
+if [ "$UEFI_ENABLE" = "yes" ] ; then
+    RECOVERY_XORRISO_OPTIONS="$RECOVERY_CONF_DIR/xorriso-options-uefi.cfg"
+else
+    RECOVERY_XORRISO_OPTIONS="$RECOVERY_CONF_DIR/xorriso-options-bios.cfg"
+fi
```

Comment:
Instead of duplicating this logic how about passing the xorriso config file as an argument to this script?  Above when calling onie-mk-iso.sh pass $(RECOVERY_XORRISO_OPTIONS) as an additional argument.